src/CMakeLists.txt

```diff
@@ -52,8 +54,27 @@ elseif(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   set(STDLIB "")
 endif()
 
-if (${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
-   set(STDLIB "${STDLIB} -static-libgcc -static-libstdc++")
+#
+# Identify build type - local or deployment (Travis)
+# the variable NUPIC_DEPLOYMENT_BUILD must be set in travis CI scripts
+#
+if($ENV{NUPIC_DEPLOYMENT_BUILD})
+  message("Doing a deployment build in TravisCI")
+else()
+  message("Doing a local build")
+endif()
+
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
This stuff isn't needed because of this:
https://github.com/numenta/nupic.core/blob/e3dfdb262fbba3158859b561c8d39f4e87d570d1/CMakeLists.txt#L27-L36